source/sdk/index.md

```diff
@@ -0,0 +1,119 @@
+---
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+layout: sdk/markdown
+title: Installing the Pebble SDK
+permalink: /sdk/
+menu_section: sdk
+menu_subsection: install
+generate_toc: true
+scripts:
+  - sdk/index
+---
+
+## Install dependencies
+
+#### MacOS
+
+You'll need Python 3.10 or above, not the MacOS default installation of 3.9. To install a newer version of Python, run:
+
+```bash
+brew install python
```

Comment:
Can we add some instructions for setting up brew as well? Just a link to https://brew.sh should be fine.